packages/omgx/wallet/README.md

```diff
@@ -97,18 +103,6 @@ $ yarn deploy #if needed - this will test and deploy the contracts, and write th
 
 The information generated during the deploy (e.g the `/deployment/local/addresses.json`) is used by the web wallet to set things up correctly. **The full test suite includes some very slow transactions such as withdrawals, which can take 100 seconds to complete. Please be patient.**
 
-### Running tests with the customized Messenger
```

Comment:
do we want the alt_messenger stuff to be on a different branch or do we want to keep this?